.gitignore

```diff
@@ -14,5 +13,5 @@ Thumbs.db
 # typedoc - for now
 docs
 
-# most logs
-**/*.log
+# code-points
+code-points.mem
```

Comment:
What is this file and what is producing it?

Reply:
https://github.com/neofinancial/mongo-memory-server-action/pull/6#issuecomment-899657481

String indication that it's a ephemeral file left around from vercel/ncc's use of code-points.